### PR TITLE
Add Create Token API

### PR DESCRIPTION
**Pull request by Ray2126**

### CODE REVIEW

**Review by subarashii-huch09 — COMMENTED** [review]
@Ray2126   Great work , looks really good 😀 💪

I have questions for the changes made in the `tokens.md` : 😃
1. In line 33, why is `ledgerId ` included in the body ?
   and what's the reason that `centrapay.nz.main` is the value for `ledgerId` ?
   
2. In the Error Responses table,  I can see the error `TOKEN_PARAMS_INVALID` is not included in the table. Is it because of security reasons? 
I can see it's documented in the cancellation reasons in Notion

Thanks 🧡

**Review by subarashii-huch09 — COMMENTED** [review]
.

File: legacy/src/api/assets/tokens.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	"type": "centrapay.token.test"
+}
+{% endjson %}
+
+{% h4 Error Responses %}
+
+| Status |              Code               |                             Description                              |
+| :----- | :------------------------------ | :------------------------------------------------------------------- |
+| 403    | TOKEN_ALREADY_CREATED           | Token with supplied parameters already exists.                       |
+| 403    | TOKEN_COLLECTION_DOES_NOT_EXIST | No collection can be found with the supplied collectionId.           |
```
Comment (daallen303) [inline legacy/src/api/assets/tokens.md:74]: This should be a generic 403 https://docs.centrapay.com/api/http-status-codes#resource-missing-or-permission-denied

File: legacy/src/api/assets/tokens.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+| 403    | ACCOUNT_DOES_NOT_EXIST          | No account can be found with the supplied accountId.                 |
```
Comment (daallen303) [inline legacy/src/api/assets/tokens.md:75]: Same here https://docs.centrapay.com/api/http-status-codes#resource-missing-or-permission-denied

File: legacy/src/api/assets/tokens.md (hunk 1 above)
Comment (Ray2126) [reply legacy/src/api/assets/tokens.md]: I've removed this from the table :)

File: legacy/src/api/assets/tokens.md (hunk 2 above)
Comment (Ray2126) [reply legacy/src/api/assets/tokens.md]: Removed as well :)

**Review by seliwe — APPROVED** [review]
Lgtm nice job :)

**Comment by subarashii-huch09** [comment]
> Lgtm nice job :)

⭐️

**Pull request merged**